announcement/models/announcement.py

```diff
@@ -103,7 +103,8 @@ def _compute_user_group_ids(self):
                 announcement.announcement_type = "user_group"
                 announcement.user_group_ids = self.env.ref("base.group_user")
             else:
-                announcement.specific_user_ids = False
```

Comment:
I think the patch may be:

```suggestion
                announcement.user_group_ids = False
```

Reply:
@pilarvargas-tecnativa thumb rule for the next time: you can't update a field in the compute of another field.

Reply:
> @pilarvargas-tecnativa thumb rule for the next time: you can't update a field in the compute of another field.

I take note, thank you